debian/README.Debian

```diff
@@ -45,3 +45,13 @@ Docker, please be sure to read the "Docker Security" article in the
 upstream documentation:
 
     https://docs.docker.com/engine/security/security/
+
+
+Importing new upstream version
+==============================
+
+After importing a new upstream version, do not forget to run
```

Comment:
This is unfortunate...  I think the least we can do here (on top of what you already documented) is to print a message when the `debian/get-orig-source.sh` script finishes to remind the developer that they still need to regenerate the manpages by hand.

Reply:
Fair enough.